pkg/kubelet/network/dns/dns.go

```diff
@@ -21,6 +21,7 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"k8s.io/kubernetes/pkg/features"
```

Comment:
nit: this package belongs to the k8s group, should be moved down.

Reply:
will reorgnize imports